### PR TITLE
Hourly FFMC data retention, past 24 hours

### PR DESCRIPTION
**Pull request by conbrad**
Adds a cronjob to run nightly at 2am to clean up hourlies from a day before yesterday

closes https://github.com/bcgov/wps/issues/3558
# Test Links:
[Landing Page](https://wps-pr-3590-e1e498-dev.apps.silver.devops.gov.bc.ca/)
[MoreCast](https://wps-pr-3590-e1e498-dev.apps.silver.devops.gov.bc.ca/morecast)
[Percentile Calculator](https://wps-pr-3590-e1e498-dev.apps.silver.devops.gov.bc.ca/percentile-calculator)
[MoreCast](https://wps-pr-3590-e1e498-dev.apps.silver.devops.gov.bc.ca/morecast)
[C-Haines](https://wps-pr-3590-e1e498-dev.apps.silver.devops.gov.bc.ca/c-haines)
[FireBat](https://wps-pr-3590-e1e498-dev.apps.silver.devops.gov.bc.ca/fire-behaviour-calculator)
[FireBat bookmark](https://wps-pr-3590-e1e498-dev.apps.silver.devops.gov.bc.ca/fire-behaviour-calculator?s=266&f=c5&c=NaN&w=20,s=286&f=c7&c=NaN&w=16,s=1055&f=c7&c=NaN&w=NaN,s=305&f=c7&c=NaN&w=NaN,s=344&f=c5&c=NaN&w=NaN,s=346&f=c7&c=NaN&w=NaN,s=328&f=c7&c=NaN&w=NaN,s=1399&f=c7&c=NaN&w=NaN,s=334&f=c7&c=NaN&w=NaN,s=1082&f=c3&c=NaN&w=NaN,s=388&f=c7&c=NaN&w=NaN,s=309&f=c7&c=NaN&w=16,s=306&f=c7&c=NaN&w=NaN,s=1029&f=c7&c=NaN&w=NaN,s=298&f=c7&c=NaN&w=NaN,s=836&f=c7&c=NaN&w=NaN,s=9999&f=c7&c=NaN&w=NaN)
[Auto Spatial Advisory (ASA)](https://wps-pr-3590-e1e498-dev.apps.silver.devops.gov.bc.ca/auto-spatial-advisory)
[HFI Calculator](https://wps-pr-3590-e1e498-dev.apps.silver.devops.gov.bc.ca/hfi-calculator)

### CODE REVIEW

File: openshift/hourlies-prune/docker/prune_hourlies.sh [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+if [ -z ${AWS_BUCKET+0} ]
+then
+    echo "AWS_BUCKET not specified"
+    echo "Specify an AWS bucket"
+    exit 1
+fi
+
+hourlies_path="sfms/uploads/hourlies"
+day_before_yesterday=$(TZ=America/Vancouver date -v-2d -I)
```
Comment (conbrad) [inline openshift/hourlies-prune/docker/prune_hourlies.sh:37]: Grabs the date in ISO 8601 and subtracts 2 days from it to get the day before yesterday

**Review by dgboss — APPROVED** [review]
LGTM!

File: openshift/hourlies-prune/README.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+# Hourlies Prune Job
+
+Deletes hourlies from object that are from the day before yesterday, to maintain the last 24 hours of hourly FFMC tifs.
```
Comment (dgboss) [inline openshift/hourlies-prune/README.md:3]:
```suggestion
Deletes hourlies from object storage that are from the day before yesterday, to maintain the last 24 hours of hourly FFMC tifs.
```

**Review by brettedw — APPROVED** [review]
Looks good!

**Pull request merged**